tests/unit/conftest.py

```diff
@@ -10,3 +13,24 @@ def root_dir(request):
 @pytest.fixture(scope="module")
 def local_env_file(root_dir):
     yield root_dir / "tests" / "test.env"
+
+
+@pytest.fixture(scope="module")
+def session_plex(root_dir):
+    """
+    Fixture to return the prometheus log exporter module.
+    """
+    # Ensure the parent directory is in the sys.path
+    path = Path(root_dir) / "birdhouse" / "optional-components" / "prometheus-log-parser" / "config" / "thredds"
```

Comment:
```suggestion
    path = root_dir / "birdhouse" / "optional-components" / "prometheus-log-parser" / "config" / "thredds"
```

`root_dir` is already a Path instance